staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -121,8 +122,6 @@ type Config struct {
 	// Will default to a value based on secure serving info and available ipv4 IPs.
 	ExternalAddress string
 
-	// SharedInformerFactory provides shared informers for resources
-	SharedInformerFactory informers.SharedInformerFactory
 	//===========================================================================
```

Comment:
These three lines should be deleted as well ?

Reply:
They belong to the next fields.